1121 services

azaroth42 opened the pull request:
Closes #1121 ; add ImageService1,2 and SearchService1 to Prezi3

azaroth42 commented:
Changed "a few" to "the following" ... sorry for making reviews stale!

jpstroop commented on source/api/presentation/3.0/index.md:
```diff
@@ -571,6 +571,29 @@ The value _MUST_ be an array of JSON objects. Each object _MUST_ have the `id` a
+| -------------- | ------------- |
+| ImageService1  | [Image API version 1][]  |
+| ImageService2  | [Image API version 2][]  |
+| SearchService1 | [Search API version 1][] |
```
Why the empty links?

azaroth42 replied:
Ahh... assistance please in getting the new link tricks to work?

jpstroop replied:
What's the trick? Something related to https://github.com/IIIF/api/pull/1261?

azaroth42 commented:
I think we need to resolve #1352 before we can merge this.

azaroth42 commented:
Up to date with the links. Also fixed lots and lots of other links to be more regular and uniquely named. Added services and links for the distinction between token/cooke/logout, and search vs autocomplete.

azaroth42 commented:
The normative / text differences are limited to http://preview.iiif.io/api/1121-services/api/presentation/3.0/#service

The rest of the changes are the link un-breaking.

jpstroop commented on source/api/presentation/3.0/index.md:
```diff
@@ -1513,7 +1540,7 @@ Recipes for enabling CORS, conditional Content-Type headers and other technical
 ## 7. Authentication
 
-It may be necessary to restrict access to the descriptions made available via the Presentation API.  As the primary means of interaction with the descriptions is by web browsers using XmlHttpRequests across domains, there are some considerations regarding the most appropriate methods for authenticating users and authorizing their access.  The approach taken is described in the [Authentication][au ...
+It may be necessary to restrict access to the descriptions made available via the Presentation API.  As the primary means of interaction with the descriptions is by web browsers using XmlHttpRequests across domains, there are some considerations regarding the most appropriate methods for authenticating users and authorizing their access.  The approach taken is described in the [Authentication][au ...
```
Style question: should XmlHttpRequests be `XmlHttpRequest`s?

azaroth42 replied:
Maybe? We don't tend to `backtick` our own class names, so I'd be at least hesitant to do it here ... but I don't think we have any consistency.

jpstroop replied:
Make a separate ticket and move on? This is easily solved in a pub.

azaroth42 replied:
Yup, 👍  Will make the issue now.

azaroth42 replied:
--> #1371
